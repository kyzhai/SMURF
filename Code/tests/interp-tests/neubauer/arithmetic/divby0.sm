a = 10 / 0

main = (a,0)$2

