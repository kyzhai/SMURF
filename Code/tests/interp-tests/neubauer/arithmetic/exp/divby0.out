PASSED SEMANTIC CHECKS
Fatal error: exception Values.Interp_error("Cannot divide by zero")
